packages/constants/src/network/network.constant.ts

```diff
@@ -33,5 +34,10 @@ export const NETWORK = {
     name: Network.AMM_DEVNET,
     server: NetworkServer.AMM_DEVNET,
     description: 'XLS-30d Automated Market Makers preview network.'
+  },
+  [Network.CUSTOM]: {
```

Comment:
Not so sure why this is needed?

Reply:
It makes the code easier because the useNetwork hook accepts a Network as parameter, which can now be custom. If I don't put this, but keep "Custom" in the enum, type check will fail